src/ontology/uberon-edit.obo

```diff
@@ -224080,41 +224046,29 @@ name: existence ends during
 def: "Relation between continuant c and occurrent s, such that every instance of c ceases to exist during some s, if it does not die prematurely." [https://orcid.org/0000-0002-6601-2165]
```

Comment:
delete def?

Reply:
or not in import?

Reply:
This definition seems to be an updated version.
In the `ro_import.owl` the definition is simple "inverse of ends with".
Maybe update definition in source?